FirstSolution/Task4Germany/Berlin.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Task4Germany
+{
+    public class Berlin
+    {
+        private static double thousandOfInhabitants = 3645.5;
+        
+        public static double ThousandOfInhabitants => thousandOfInhabitants;
+
+        public static void Main(string[] args)
```

Comment:
Why do you need this Main method here?

Reply:
I've made it because Rider can't use classes from another namespaces without main methods.

Reply:
Sounds strange, investigate it please

Reply:
Changed